### ISSUE TITLE
editing unit stats. when?

### ISSUE THREAD

**Issue by ilosevsrat** [Sh0wdown/UnofficialCrusaderPatch#478]
so, when we be able to edit unit stats ? like HP pool or damage :)

i strongly believe Swordsmen and pikeman need HP increase.

**Comment by GRhin** [comment]
This has been raised several times, and showdown has hinted that he might be able to do a settings file like the aic, however he hasn't found all the stats locations yet.

As for buffing swords and pikes, I agree some balancing is needed, I just never thought those were underpowered. Can I ask your reasoning? Maybe nerfing another unit a little would fix this issue as well.

 My opinion is that shields need to walk at pikemen speed - this will allow other melee to catch them thereby reducing their effectiveness and increasing utility of melee units, and maybe reduce price of Arabian archers or increase horse archers (their prices are too similar for me) but that's just a nice to have.

**Comment by ilosevsrat** [comment]
> This has been raised several times, and showdown has hinted that he might be able to do a settings file like the aic, however he hasn't found all the stats locations yet.
> 
> As for buffing swords and pikes, I agree some balancing is needed, I just never thought those were underpowered. Can I ask your reasoning? Maybe nerfing another unit a little would fix this issue as well.
> 
> My opinion is that shields need to walk at pikemen speed - this will allow other melee to catch them thereby reducing their effectiveness and increasing utility of melee units, and maybe reduce price of Arabian archers or increase horse archers (their prices are too similar for me) but that's just a nice to have.

from my opinion, crossbowman are too powerfull, 200 CB can stop 400+ swordsman army easy..

**Comment by Truetobi91** [comment]
Make Spears great again!

I mean it's just hillarious that a group of spears cannot outdamage the respawn rate of the villagers. 

I'd go with this: 
Arrow Resistance: Check (already in the UCP)
Walking Speed: Always Running (just like Archers; both have no Armor)
Damage: Increased against Villagers/Workers

**Comment by GRhin** [comment]
Crossbows are the counter to armoured units, it makes sense that they are strong against them, but there are other ways to kill off crossbows before committing 400 swords... any ranged unit with shields will draw fire away so your melee units get in undamaged. Catapults and trebs kill any towers the cows are on, and if you dont have shields then HA will kill them easy as long as they keep moving and changing directions, if they aren't on a wall then diseased cows will slaughter them easy.

**Comment by J-T-de** [comment]
Yeah, I think there are many taste-dependent changes one could make.
In my opinion: 
- Pikeman slightly slower, I view them as defensive unit
- Swordsman stronger (I know, historically the Pikeman was a counter for swordsman, but come on, swordsman should be way more badass!)
- shield speed and health reduction, no possibility to put them on towers
- Crossbowman nerf (more damage per shot, but way lower rate of fire (roughly same DMG/min) so swarming gets more effective? Less firing range? Just something to promote not to only spam Crossbowman and give Bowman some reason to exist)
...

Anyway, back to the main topic: When its done, its done (also see https://github.com/Sh0wdown/UnofficialCrusaderPatch/issues/472)

**Comment by ilosevsrat** [comment]
> Yeah, I think there are many taste-dependent changes one could make.
> In my opinion:
> 
> * Pikeman slightly slower, I view them as defensive unit
> * Swordsman stronger (I know, historically the Pikeman was a counter for swordsman, but come on, swordsman should be way more badass!)
> * shield speed and health reduction, no possibility to put them on towers
> * Crossbowman nerf (more damage per shot, but way lower rate of fire (roughly same DMG/min) so swarming gets more effective? Less firing range? Just something to promote not to only spam Crossbowman and give Bowman some reason to exist)
>   ...
> 
> Anyway, back to the main topic: When its done, its done (also see #472)

crossbowman need slower rate of fire for sure, but  decreasing his range will ruin ai that uses CB only.

**Comment by J-T-de** [comment]
> > Yeah, I think there are many taste-dependent changes one could make.
> > In my opinion:
> > 
> > * Pikeman slightly slower, I view them as defensive unit
> > * Swordsman stronger (I know, historically the Pikeman was a counter for swordsman, but come on, swordsman should be way more badass!)
> > * shield speed and health reduction, no possibility to put them on towers
> > * Crossbowman nerf (more damage per shot, but way lower rate of fire (roughly same DMG/min) so swarming gets more effective? Less firing range? Just something to promote not to only spam Crossbowman and give Bowman some reason to exist)
> >   ...
> > 
> > Anyway, back to the main topic: When its done, its done (also see #472)
> 
> crossbowman need slower rate of fire for sure, but decreasing his range will ruin ai that uses CB only.

If the range of a crossbowman on a tower equals the range of the bowman on plain field I guess it could work. I think the Pig and the Sherif would not suffer too much from this, and I think their offensive capabilities lie more in their maceman and not in their crossbowman. Maybe the crossbowman on patrol will suffer, but I guess, if such an option exists, one would notice it fast whether or not it works.

**Comment by Mizo5119** [comment]
Here are changes that I view as necessary:

-Nerfing Shields to hell, reducing their movement speed if possable and increasing gold cost to 50 so that they wont be slammed. I d even go for health increase against missiles if those 2 are adjusted.

-Increasing the price of Horse Archers from 80 to 100/110 gold
 Increasing cost of Assassins to 90/100 gold as well. ( It seems they are the most used pvp unit, so might as well make them taxing. Would make using alternatives better)
- Reducing cost of Arab Archers from 75 to 60/65. They're far more expensive than they're worth. Purchasing Bows and building archers is cheaper than even using fletcher. Their stat differences dont warrant the huge disparity in their cost efficiency.
